src/orb-release/cli.js

```diff
@@ -0,0 +1,184 @@
+            type = releaseTypes.indexOf('patch');
+          }
+          if (/^feat\(?.*\)?:/.test(commit.subject) && type < releaseTypes.indexOf('minor')) {
+            type = releaseTypes.indexOf('minor');
+          }
+          if (/.*(BREAKING CHANGE).*/.test(commit.message)) {
+            type = releaseTypes.indexOf('major');
+          }
+          return true;
+        }
+        return false;
+      });
+  } else {
+    logger.warn('Unable to determine last release date');
+  }
+  logger.log(type ? 'Proposed release type: %s' : 'No release', releaseTypes[type]);
+  return releaseTypes[type];
+}
+
+function getOrbs() {
+  const orbs = [];
+  const dirs = fs.readdirSync(config.orbDir)
+    .filter((dir) => /^[A-Za-z0-9_-]+$/.test(dir));
+  dirs.forEach((orb) => {
+    const dir = `${config.orbDir}/${orb}`;
+    orbs.push({
+      name: orb,
+      dir,
+      src: `${dir}/${config.orbSrc}`,
+      tmp: fs.mkdtempSync(`/tmp/${orb}-`),
+    });
+  });
+  return orbs;
+}
+
+function diffOrb(orb) {
+  logger.log('Checking orb %s', orb.name);
+  let ret = false;
+  const pubFile = `${orb.tmp}/src.yml`;
+  shell.exec(`~/circleci orb source ${config.namespace}/${orb.name} > "${pubFile}"`);
+  if (shell.error()) {
+    return ret;
+  }
+  const pubSource = fs.readFileSync(pubFile).toString('utf-8');
+  const locSource = `${fs.readFileSync(orb.src).toString('utf-8')}\n`;
+  const patch = diff.diffTrimmedLines(pubSource, locSource);
+  patch.forEach((entry) => {
+    if (entry.added || entry.removed) {
+      ret = true;
+    }
+  });
+  logger.log(ret
+    ? '  Orb source changed, proceeding with release'
+    : '  Orb source unchanged, skipping release');
+  return ret;
+}
+
+function releaseOrb(orb, type) {
+  return shell.exec(`~/circleci orb publish increment "${orb.src}" ${config.namespace}/${orb.name} ${type}`);
+}
+
+async function releaseOrbs(opts) {
+  config = {
+    ...opts,
+    ...config,
+  };
+  const releaseType = await getReleaseType();
+  if (releaseType) {
+    getOrbs().forEach((orb) => {
+      if (diffOrb(orb)) {
+        releaseOrb(orb, releaseType);
+      }
+    });
+  }
+  logger.log('Done');
+}
+
+function baseargs(y) {
+  return y
+    .option('token', {
+      type: 'string',
+      alias: 'auth',
+      describe: 'CircleCI CLI Token (or env $CIRCLECI_CLI_TOKEN)',
+      required: true,
+    })
+    .option('namespace', {
+      type: 'string',
+      describe: 'The namespace to publish the orbs under',
+      default: 'adobe',
+      required: false,
+    })
+    .option('orb-dir', {
+      type: 'string',
+      alias: 'orbDir',
+      describe: 'The relative path to the orbs',
+      default: './.circleci/orbs',
+      required: false,
+    })
+    .option('orb-src', {
+      type: 'string',
+      alias: 'orbSrc',
+      describe: 'The name of the orb source file',
+      default: 'orb.yml',
+      required: false,
+    })
+    .option('changelog', {
```

Comment:
Why `changeLog` and not `change-log` ? (same pattern than other options)

Reply:
because the file is typically called `CHANGELOG.md`